src/components/message/message.twig

```diff
@@ -0,0 +1,21 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
+
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} class="c-message{% if class %} {{class}}{% endif %}">
+  {% block content %}
+  <p>
+    {{label | default('Hi, I\'m a message!')}}
+  </p>
+  {% endblock %}
+  {% if includeClose %}
+  <button aria-label="close" class="c-message__close u-pad-none" type="button">
+    {% include '../../assets/icons/close.svg.twig' with {
+        class: 'c-icon'
+      } %}
```

Comment:
@Paul-Hebert Do you remember if it's best to use `aria-hidden="true"` or `role="presentation"` on icons in buttons like this?

Reply:
If we want this to be hidden from screen readers we should use `aria-hidden`. 

My understanding is that `role="presentation"` tells a screen reader “ignore the semantics of this element” while `aria-hidden="true"` says “ignore this completely.”

https://timwright.org/blog/2016/11/19/difference-rolepresentation-aria-hiddentrue/